plugins/linguist-backend/src/api/LinguistBackendApi.ts

```diff
@@ -67,6 +68,7 @@ export class LinguistBackendApi {
     batchSize?: number,
     useSourceLocation?: boolean,
     kind?: string[],
+    linguistJsOptions?: Record<string, any>,
```

Comment:
```suggestion
    linguistJsOptions?: Record<string, unknown>,
```

We prefer `unknown` over `any` so let's update it here and in PluginOptions

Reply:
Done